INSTALL.md

```diff
@@ -1065,6 +1065,13 @@ Build with support for the integrated tracing api.
 
 See manual pages OSSL_trace_set_channel(3) and OSSL_trace_enabled(3) for details.
 
+### enable-sslkeylog
+
+Build with support for the SSLKEYLOGFILE environment variable
+
+When enabled, setting SSLKEYLOGFILE to a file path records the keys exchanged
+during a TLS handshake for use in analysis tools like wireshark.
```

Comment:
Do we want to mention any security risks here? Note there is an extensive discussion of the security risks in the draft

Reply:
I would be happy with something like: `Those tools then can decrypt application payload found in captured packets using keys from key log file` See Section 3 Security consideration in [draft-thomson-tls-keylogfile](https://datatracker.ietf.org/doc/draft-thomson-tls-keylogfile/)